data/sql/updates/pending_db_world/rev_1624636115436386300.sql

```diff
@@ -0,0 +1,52 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1624636115436386300');
+
+DELETE FROM `gossip_menu_option_locale` WHERE `MenuID` = 10189, OptionID = 0 AND `Locale` IN ('esES','esMX');
+INSERT INTO `gossip_menu_option_locale` (`MenuID`, `OptionID`, `Locale`, `OptionText`, `BoxText`) VALUES
+(10189,0,'esES','Lady Valiente, estoy listo para ir a Orgrimmar. Abre el portal.','');
+(10189,0,'esMX','Lady Valiente, estoy listo para ir a Orgrimmar. Abre el portal.','');
+
+
+DELETE FROM `creature_text_locale` WHERE `CreatureID` IN (32364, 32346) AND `Locale` IN ('esES','esMX');
```

Comment:
I think it would be good to specify the ID, because today the dialog may not exist, but it is a good practice to delete only what we add.